people/views.py

```diff
@@ -70,6 +70,28 @@ def profile(request, year, pk):
 		'roles': RecruitmentApplication.objects.filter(user = user, status = 'accepted').order_by('recruitment_period__fair')
 	})
 
+def profile_delete(request, year, pk):
+	fair = get_object_or_404(Fair, year = year)
+	user = get_object_or_404(User, pk = pk)
+
+	profile = Profile.objects.filter(user = user).first()
+	if not profile: profile = Profile.objects.create(user = user, no_dietary_restrictions = False)
+	
+	application = RecruitmentApplication.objects.filter(user = user, status = 'accepted', recruitment_period__fair = fair).first()
```

Comment:
Not sure how this can work... 
1. You first retrieve a profile.
2. Checks if profile is empty.
  2.1. If True, create a profile
3. Retrieve a RecruitmentApplication  where status is `accepted`.

If a user don't have a profile that automatically means that you won't have an recruitment application...

And the `application`-variable isn't used until row 91, which means you can drop the `application = RecruitmentApplication.objects.filter(user = user, status = 'accepted', recruitment_period__fair = fair).first()` to above 87, above the `return TemplateResponse(...`.